packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -955,14 +1031,15 @@ export class StatsAnalyzer extends EventsScope {
   private processInboundRTPResult(result: any, mediaType: any) {
     const sendrecvType = STATS.RECEIVE_DIRECTION;
 
+    this.processTrackResult(result, mediaType, sendrecvType);
     if (result.bytesReceived) {
       let kilobytes = 0;
       const receiveSlot = this.receiveSlotCallback(result.ssrc);
       const idAndCsi = receiveSlot
         ? `id: "${receiveSlot.id || ''}"${receiveSlot.csi ? ` and csi: ${receiveSlot.csi}` : ''}`
         : '';
 
-      if (result.frameWidth && result.frameHeight) {
+      if (result.frameWidth && result.frameHeight && !mediaType.includes('audio')) {
```

Comment:
same as above, there are many more video specific things further below (like pliCount)

Reply:
Removed this.